packages/shared/lib/transactions.ts

```diff
@@ -32,7 +32,10 @@ export type TokenDisplayBalances = Omit<TokenBalances, 'balance' | 'lockedBalanc
   lockedBalance: bigint
 }
 
-export type AssetInfo = TokenInfo & { verified?: boolean }
+export type AssetInfo = Omit<TokenInfo, 'decimals'> & {
+  decimals?: number
```

Comment:
You will have probably answered my above questions by now, I just write this comment for me